eslint.config.js

```diff
@@ -997,6 +997,7 @@ export default tseslint.config(
 				{
 					'target': 'src/vs/workbench/api/~',
 					'restrictions': [
+						'@c4312/eventsource-umd',
```

Comment:
This is just a fork of `eventsource` with a UMD build. Seems like we _must_ have this to load it on all platforms.